src/Parse/Syntax/FieldParser.php

```diff
@@ -377,17 +377,36 @@ protected function processTagsRegex($string, $tags)
     /**
      * Splits an option string to an array.
      *
-     * one|two           -> [one, two]
-     * one:One|two:Two   -> [one => 'One', two => 'Two']
+     * one|two               -> [one, two]
+     * one:One|two:Two       -> [one => 'One', two => 'Two']
+     * \Path\To\Class|method -> \Path\To\Class::method()
```

Comment:
@JaxWilko is this docblock up to date with the latest format?

Reply:
@LukeTowers ah no it's not been updated. Should read `\Path\To\Class::method -> \Path\To\Class::method(): array` I will push an update when I get on my main PC.